help_function/src/onedpl_test_group_load.cpp

```diff
@@ -0,0 +1,195 @@
+// ====------ onedpl_test_group_load.cpp------------ *- C++ -* ----===//
+
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//
+// ===----------------------------------------------------------------------===//
+#include <sycl/sycl.hpp>
+#include <dpct/dpct.hpp>
+#include <dpct/dpl_utils.hpp>
+#include <iostream>
+#include <oneapi/dpl/iterator>
+
+
+template<dpct::group::load_algorithm T>
+bool helper_validation_function(const int* ptr, const char * func_name){
+  if ( T == dpct::group::load_algorithm::BLOCK_LOAD_DIRECT)

```

Comment:
Unimportant, but this can be `if constexpr (T == dpct::group::load_algorithm::BLOCK_LOAD_DIRECT)`